easybuild/easyconfigs/r/RELION/RELION-3.1.3-fosscuda-2020b.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# modified from https://github.com/FredHutch/easybuild-life-sciences/blob/main/fh_easyconfigs/r/RELION/RELION-3.1.2-fosscuda-2020b.eb
+
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '3.1.3'
+
+github_account = '3dem'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s'
+description = """RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer
+ program for Maximum A Posteriori refinement of (multiple) 3D reconstructions or 2D class
+ averages in cryo-electron microscopy."""
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+toolchainopts = { 'openmp' : True }
+
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['f10934058e0670807b7004899d7b5103c21ad77841219352c66a33f98586d42e']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('X11', '20201008'),
+    ('LibTIFF', '4.1.0'),
+    ('FLTK', '1.3.5'),
+    ('ctffind', '4.1.14'),
+    ('tbb', '2020.3'),
+    ('Ghostscript', '9.53.3')
+]
+
+hiddendependencies = [
+    ('LibTIFF', '4.1.0'),
```

Comment:
Whether dependencies are hidden or not is a site-specific aspect, it should not be hardcoded in the easyconfig file, so this should be stripped out in order to make this PR eligible for merging.

A better approach is to configure EasyBuild with `--hide-deps=LibTIFF` locally.

Reply:
Works also fine. Will push the updated eb file